src/app/app.module.js

```diff
@@ -0,0 +1,55 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('esn-frontend-account-inbox', [
+       
+        'op.dynamicDirective',
+        'esn.session',
+        'esn.configuration',
+        'esn.async-action',
+        'esn.i18n'
+       
+    ]);
+})(angular);
+
+require('esn-frontend-common-libs/src/frontend/components/dynamic-directive/src/dynamic-directive.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/session.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/async-action.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/i18n/i18n.module.js');
+
+
+require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/services/common/inbox-restangular.service.js')
+require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/constants.js');
```

Comment:
1. If you need a service from another Angular module (in this case is linagora.esn.unifiedinbox), you should require its app.module.js where defines the module instead of the service.
2. But there is a huge number of unnecessary files if we require everything from `linagora.esn.unifiedinbox` module. So, you need to create a new Angular module in `esn-frontend-inbox` SPA to wrap the common service: `esn.inbox.libs` for example